model/fv_control.F90

```diff
@@ -445,22 +445,11 @@ subroutine fv_control_init(Atm, dt_atmos, this_grid, grids_on_this_pe, p_split)
 
      call set_namelist_pointers(Atm(this_grid))
      call fv_diag_init_gn(Atm(this_grid))
-#ifdef INTERNAL_FILE_NML
-     if (this_grid .gt. 1) then
-        write(Atm(this_grid)%nml_filename,'(A4, I2.2)') 'nest', this_grid
-        if (.not. file_exists('input_'//trim(Atm(this_grid)%nml_filename)//'.nml')) then
-           call mpp_error(FATAL, "Could not find nested grid namelist "//'input_'//trim(Atm(this_grid)%nml_filename)//'.nml')
-        endif
-     else
-        Atm(this_grid)%nml_filename = ''
-     endif
-     call read_input_nml(Atm(this_grid)%nml_filename) !re-reads into internal namelist
-#endif
+     call read_input_nml(alt_input_nml_path=Atm(this_grid)%nml_filename) !re-reads into internal namelist
      call read_namelist_fv_grid_nml
      call read_namelist_fv_core_nml(Atm(this_grid)) ! do options processing here too?
-     call read_namelist_test_case_nml(Atm(this_grid)%nml_filename)
      !TODO test_case_nml moved to test_cases
```

Comment:
Sorry to be picky here, but I don't think we need this TODO comment anymore.

Reply:
I don't mind the extensive review - I want to draw attention to these other lines of code in fv_control.  Would you like me to also remove these comments? 

https://github.com/NOAA-GFDL/GFDL_atmos_cubed_sphere/blob/29396d396bcf57e1121f16d90f29c2d85903ed49/model/fv_control.F90#L57-L58
https://github.com/NOAA-GFDL/GFDL_atmos_cubed_sphere/blob/29396d396bcf57e1121f16d90f29c2d85903ed49/model/fv_control.F90#L1053

Reply:
It makes sense to clean those up too.